src/frontend/state/GlobalState.tsx

```diff
@@ -92,9 +92,9 @@ export class GlobalState extends PureComponent<Props> {
       []
     for (const igame in games) {
       for (const installedGame of installedGames) {
-        if (installedGame.appName === games[igame].app_name) {
-          games[igame].install = installedGame
-          games[igame].is_installed = true
+        if (installedGame.appName === games[igame]!.app_name) {
```

Comment:
should not be an interrogation mark instead of a exclamation one to see if the item is undefined?

Reply:
We're looping over all indexes in `games` (see the `for (const igame in games)` above). Thus, it's not possible for `games[igame]` to be undefined here